### PR TITLE
Link data and style dirs from nb dir

### PR DESCRIPTION
**Pull request by jni**
Partial fix (*NIX only) for #319

---
This change is [Reviewable](https://reviewable.io/reviews/elegant-scipy/elegant-scipy/320)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -119,6 +119,8 @@ nbs: $(addsuffix .ipynb,$(NBS_))
 # build_dirs: directories for build products
 build_dirs:
 	mkdir -p $(BUILD_HTML) $(BUILD_NB) $(BUILD_HTMLBOOK) $(FIGURES)
+	ln -s data $(BUILD_NB)/data
```
Comment (stefanv) [inline Makefile]: Can fail if symlink already exists. ln -sf?

**Comment by jni** [comment]
Actually @stefanv it was broken anyway, turns out symlinks are interpreted relative to the directory where the link lives, did you know that??? ie it should have been `ln -s ../data ipynb/data`. This book has been quite the trip. =D

Anyway, I don't want to clobber user files, so I've written my own little Python symlinker.

**Comment by jni** [comment]
Did notedown get updated or something? Dunno what's wrong with the wercker build...

**Comment by jni** [comment]
Nope, building works fine on my machine. Dunno what the deal is but discovered a flaw in the Makefile if trying to build a single chapter, so fixed that too.

**Comment by stefanv** [comment]
It's incredible that the GNU tools do not have a "ignore failure" flag?

**Pull request merged**